apps/web/src/components/terminal/SessionStatusDisplay.tsx

```
@@ -75,6 +75,18 @@ export function SessionStatusDisplay({ session, gitBranch }: SessionStatusDispla
         </div>
       )}
 
+      {/* Skip-permissions indicator */}
+      {session.skipPermissions && (
+        <div
+          className="flex items-center gap-1 text-xs text-amber-400 shrink-0 px-1.5 py-0.5 rounded bg-amber-500/10"
+          title="Running with skip-permissions mode"
+          aria-label="Running with skip-permissions mode"
+        >
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)